install-guides/README.md

```diff
@@ -57,36 +26,41 @@ In kubeadm these variables can be set in a file that sits in
  Replace the line KUBELET_EXTRA_ARGS with the following:
 
 ```
-KUBELET_EXTRA_ARGS="\
+KUBELET_EXTRA_ARGS=\
 --cpu-manager-policy=static \
 --kube-reserved=cpu=1,memory=2Gi,ephemeral-storage=1Gi \
 --system-reserved=cpu=1,memory=2Gi,ephemeral-storage=1Gi \
---feature-gates=TopologyManager=true  --topology-manager-policy=single-numa-node"
+--topology-manager-policy=single-numa-node \
+--feature-gates="TopologyManager=true,KubeletPodResourcesGetAllocatable=true"
```

Comment:
```TopologyManager``` enabled by default starting with K8S ver 1.18 
https://kubernetes.io/docs/tasks/administer-cluster/topology-manager/#enable-the-topology-manager-feature

Reply:
👍